src/solvers/flattening/functions.h

```diff
@@ -37,7 +39,7 @@ class functionst
   }
 
 protected:
-  prop_convt &prop_conv;
+  decision_proceduret &decision_procedure;
```

Comment:
It's strange to see a file live under `flattening` but then not actually use the propositional variant of a decision procedure.

Reply:
It really is a 'lowering', and should move there.

Reply:
Now moved.